src/test/ui/argument-suggestions/extra_arguments.stderr

```diff
@@ -47,28 +47,26 @@ error[E0059]: multiple arguments to this function are incorrect
   --> $DIR/extra_arguments.rs:11:14
    |
 LL |   one_arg(1, "", 1.0);
-   |              ^^  ^^^ no parameter of type {float} is needed one_arg
-   |              |
-   |              no parameter of type &'static str is needed one_arg
+   |            --^^--^^^
+   |            | |   |
+   |            | |   this parameter of type {float} isn't needed for one_arg
+   |            | this parameter of type &'static str isn't needed for one_arg
+   |            help: removing these arguments may help
```

Comment:
```suggestion
   |            help: remove these arguments
```
It would be *really* nice to give a structured suggestion: `one_arg(1);`, but I understand if that's tricky.

Reply:
as far as I can tell, if you only provide one suggestion, it renders it inline like this, but if you provide more than one, it renders as a separate block.  So this is showing up this way because of my refactor to handle commas correctly I'm deleting multiple arguments in one block, rather than each individual block.  I might split it up again, the logic for handling the commas correctly is just tricky lol.

Reply:
consider using `span_suggestion_verbose` to make it always render on its own subwindow

Reply:
There's no multipart_suggestion_verbose; should I add it? Should I record each change as a separate suggestion?

Reply:
I think that `multipart_suggestion`s are _always_ verbose (there is no way to display them without them being shown in a span label unless there's a single suggestion... at which point I think the inline output would be desired).